src/crate/theme/rtd/conf/__init__.py

```diff
@@ -128,12 +129,28 @@
 }
 
 # Configure OpenGraph extension
-ogp_site_url = "https://crate.io/"
-ogp_description_length = 150
-ogp_site_name = "CrateDB documentation"
-ogp_image = "https://crate.io/hubfs/static/og_crateio.jpg"
+#
+# About text lengths.
+#
+# Original documentation says:
+# - ogp_description_length
+#   Configure the amount of characters taken from a page. The default of 200 is probably good
+#   for most people. If something other than a number is used, it defaults back to 200.
+#   -- https://sphinxext-opengraph.readthedocs.io/en/latest/#options
+#
+# Other people say:
+# - og:title 40 chars
+# - og:description has 2 max lengths:
+#   When the link is used in a Post, it's 300 chars. When a link is used in a Comment, it's 110 chars.
+#   So you can either treat it as 110, or, write your Descriptions to 300 but make sure the first 110
+#   is the critical part and still makes sense when it gets cut off.
+#   -- https://stackoverflow.com/questions/8914476/facebook-open-graph-meta-tags-maximum-content-length
+ogp_site_url = "https://crate.io/docs/"
+ogp_description_length = 300
+ogp_site_name = "CrateDB Documentation"
+ogp_image = "https://crate-docs-theme.readthedocs.io/en/latest/_static/images/cratedb-logo-h400.png"
```

Comment:
shouldn't we host that image on the other thing, so it a) comes from the crate.io domain and b) can replaced without editing the docs theme?

Reply:
It's not bad to have it self-contained either.